src/components/Popup/Popup.scss

```diff
@@ -1,4 +1,37 @@
-// placeholder
-.wkp-popup {
-  display: block;
+.tm-popup {
+  div:first-child {
+    -webkit-animation: fadein 0.5s; /* Safari, Chrome and Opera > 12.1 */
+    -moz-animation: fadein 0.5s; /* Firefox < 16 */
+    -ms-animation: fadein 0.5s; /* Internet Explorer */
+    -o-animation: fadein 0.5s; /* Opera < 12.1 */
```

Comment:
Autoprefixer takes care of adding this for you so no need to write css with prefix.